src/api/routes/index.js

```diff
@@ -3,13 +3,15 @@ import authRouter from './authRouter';
 import resetRouter from './password.reset';
 import articleRouter from './articleRouter';
 import profileRouter from './profileRouter';
-import authenticate from '../../middlewares/authenticate';
```

Comment:
@truestbyheart why have you removed this middleware?

Reply:
@alainburindi didn't use @MCFrank16 middleware, and I was supposed to implement this feature on his route so to avoid conflict I had to move everything to ``profileRouter``. @alainburindi said he will relay this to you.